index.js

```diff
@@ -2,7 +2,8 @@
 
 const fs = require('hexo-fs');
 const path = require('path');
-const Prism = require('./prism.js');
+const Prism = require('prismjs');
+require('prismjs/plugins/line-numbers/prism-line-numbers.min');
```

Comment:
line-numbers require statement is necessary to load the plugin in to context. Without that line, line numbers are not working.